FlorenceBML/BMLacq679/BMLacq679.xml

```diff
@@ -35,6 +36,135 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                        <item xml:id="q4" n="4">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="31r" to="40v"/>
+                                        </item> 
+                                        <item xml:id="q5" n="5">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="41r" to="50v"/>
+                                        </item> 
+                                        <item xml:id="q6" n="6">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="51r" to="60v"/>
+                                        </item> 
+                                        <item xml:id="q7" n="7">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="61r" to="70v"/>
+                                        </item> 
+                                        <item xml:id="q8" n="8">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="71r" to="80v"/>
+                                        </item> 
+                                        <item xml:id="q9" n="9">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="81r" to="90v"/>
+                                        </item> 
+                                        <item xml:id="q10" n="10">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="91r" to="100v"/>
+                                        </item> 
+                                        <item xml:id="q11" n="11">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="101r" to="110v"/>
+                                        </item> 
+                                        <item xml:id="q12" n="12">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="111r" to="120v"/>
+                                        </item> 
+                                        <item xml:id="q13" n="13">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="121r" to="130v"/>
+                                        </item> 
+                                        <item xml:id="q14" n="14">
+                                            <dim unit="leaf">4</dim>
+                                            <locus from="131r" to="134v"/>
+                                        </item> 
+                                        <note>Quire marks with Ethiopian numbers on the top left.</note>
```

Comment:
This information could better be included in `<signatures>`, like in this example: 
```
  <collation>
            <signatures>Undecorated quire marks are written in the main hand, in the middle of the upper margin of <locus target="#3r"></locus>,
              <locus target="#11r"></locus> ….</signatures>
```

Reply:
Done.